src/sonic-config-engine/sonic-cfggen

```diff
@@ -137,6 +137,13 @@ def ip_network(value):
         return "Invalid ip address %s" % value
     return r_v.network
 
+def load_namespace_config(asic_name):
+    if not SonicDBConfig.isInit():
+        if is_multi_asic():
+            SonicDBConfig.load_sonic_global_db_config(namespace=asic_name)
+        else:
+            SonicDBConfig.load_sonic_db_config()
```

Comment:
>load_sonic_db_config [](http://example.com/codeflow?start=26&length=20)

Could you use `SonicDBConfig.initialize`? yours is for backward-compatible purpose only, should be deprecated in new code.

Reply:
Changed.